src/app/psat/help-panel/pump-operations-help/pump-operations-help.component.html

```diff
@@ -1,93 +1,123 @@
+      electricity/fuel options are changed.
+      Emission Output Rates were obtained from the EPA and the Electronic Code of Federal Regulations.
+      <hr>
+      <div class="my-2">
         <h6>
-            CO<sub>2</sub> Savings 
+          <small class="text-muted">
+            <a href="https://www.epa.gov/energy/emissions-generation-resource-integrated-database-egrid"
+              target="_blank">
+              US EPA Emissions & Generation Resource Integrate Database (eGRID)</a>
             <br>
-            <small class="text-muted">
-                This calculator lets the user determine carbon dioxide (CO<sub>2</sub>) emissions for given
-                electricity and fuel uses. Enter the facility’s region for electricity or the fuel type and the electricity use
-                or fuel energy. The emission output rate can be edited, but will return to the default value when the
-                electricity/fuel options are changed.
-                Emission Output Rates were obtained from the EPA and the Electronic Code of Federal Regulations.
-                <hr>
-                <div class="my-2">
-                    <h6>
-                      <small class="text-muted">
-                        <a href="https://www.epa.gov/energy/emissions-generation-resource-integrated-database-egrid" target="_blank">
-                          US EPA Emissions & Generation Resource Integrate Database (eGRID)</a>
-                        <br>
-                        <hr>
-                        <a href="https://www.epa.gov/sites/production/files/2015-07/documents/emission-factors_2014.pdf" target="_blank">
-                          US EPA Emissions Factors for Greenhouse Gas Inventories (PDF)</a>
-                        <br>
-                        <hr>
-                        <a href="https://www.ecfr.gov/cgi-bin/text-idx?SID=5918762f070a5581dfde66942e2e91e6&mc=true&node=ap40.23.98_138.1&rgn=div9"
-                          target="_blank">
-                          e-CFR Default CO<sub>2</sub> Emission Factors for Various Types of Fuel</a>
-                      </small>
-                    </h6>
-                  </div>
-            </small>
-            
+            <hr>
+            <a href="https://www.epa.gov/sites/production/files/2015-07/documents/emission-factors_2014.pdf"
+              target="_blank">
+              US EPA Emissions Factors for Greenhouse Gas Inventories (PDF)</a>
+              <hr>
+          </small>
         </h6>
-    </div>
-
+      </div>
+    </small>
+  </div>
 
+  <div class="my-2" *ngIf="currentField == 'eGridSubregion'">
+    <h6>
+      eGrid Sub-Region
+      <small class="text-muted">
+        <br>
+        eGrid sub-region where electricity is generated.
+        <hr>
+      </small>
+    </h6>
+    <img src="https://www.epa.gov/sites/default/files/2021-02/2019_egrid_subregions.png" alt="eGridSubRegions">
```

Comment:
Can you download and save this image to the project instead of referencing and external link please

Reply:
place in assets/images/ folder